src/test/java/org/test/device/DeviceApplicationTests.java

```diff
@@ -0,0 +1,40 @@
+package org.test.device;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Map;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+/**
+ * Created by Alireza on 1/6/2019.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(properties = {"management.port=0"})
+public class DeviceApplicationTests {
+    @Value("${local.management.port}")
+    private int mgt;
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
+
+    @Test
+    public void shouldReturn200WhenSendingRequestToManagementEndpoint() throws Exception {
+        @SuppressWarnings("rawtypes")
+        ResponseEntity<Map> entity = this.testRestTemplate.getForEntity(
```

Comment:
This is a `get` request but I can only see a PostMapping in your controller. Does this test pass?

Reply:
Yes, this test passes when the application is correctly started. Actually it's a get request to the "actuator" endpoint. As I've coded integration tests when an application is correctly running the test will be passed.